src/rgw/motr/synchronization/rgw_motr_sync.h

```diff
@@ -0,0 +1,70 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=2 sw=2 expandtab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * SAL implementation for the CORTX Motr backend
+ *
+ * Copyright (C) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ *
+ */
+#ifndef __MOTR_SYNCHROZATION__
+#define __MOTR_SYNCHROZATION__
+
+#include <unistd.h>
+
+#include "include/types.h"
+#include "include/utime.h"
+#include "rgw_sal_motr.h"
+
+struct Motr_locker_info_t
```

Comment:
either we should use a snake case or camel case, so it should be either `MotrLockerInfoType` or `motr_locker_info_t`.

Reply:
snake case is used and it is used in  Ceph's code